frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_projects.py

```diff
@@ -181,28 +181,35 @@ def get(self, query, **kwargs):
         return {"items": projects, "meta": paginator.meta}
 
 
-@apiv3_projects_ns.route("/add/<ownername>")
+@apiv3_projects_ns.route("/add/<ownername>/")
```

Comment:
Is this a breaking API change?  Note that some clients do not necessarily have to follow redirects.

Reply:
Sorry, this is a typo. I will revert this back.